p/scripts/main.js

```diff
@@ -697,7 +697,7 @@ function auto_share(key) {
 	if (!share) {
 		return;
 	}
-	const shares = share.parentElement.querySelectorAll('.dropdown-menu .item a');
+	const shares = share.parentElement.querySelectorAll('.dropdown-menu .item [data-type]');
```

Comment:
```suggestion
	const shares = share.parentElement.querySelectorAll('.dropdown-menu .item a, .dropdown-menu .item button');
```

Reply:
Another idea could be to add a `share-action` class on the `<button>` and `<a>` in `\app\views\helpers\index\normal\entry_bottom.phtml`. It would make the JavaScript line better: 
const shares = share.parentElement.querySelectorAll('.dropdown-menu .item.share .share-action');

Reply:
You could write it as `.dropdown-menu .item.share :is(a, button)` nowadays.

Reply:
yes, but I do not like to travers on tags, I prefer to travers ids and classes

Reply:
I think the original proposal with `[data-type]` does that better than adding a classname. `[data-x]` is clearly meant for JS and not for styling.